lib/src/test/kotlin/starknet/account/StandardAccountTest.kt

```diff
@@ -302,6 +302,30 @@ class StandardAccountTest {
         assertTrue(receipt.isAccepted)
     }
 
+    @ParameterizedTest
+    @MethodSource("getAccounts")
+    fun `sign and send declare v2 transaction (cairo compiler v2)`(accountAndProvider: AccountAndProvider) {
+        val (account, provider) = accountAndProvider
+        val contractCode = Path.of("src/test/resources/compiled_v2/${provider::class.simpleName}_new_syntax_test_contract.json").readText()
+        val casmCode = Path.of("src/test/resources/compiled_v2/${provider::class.simpleName}_new_syntax_test_contract.casm").readText()
+
+        val contractDefinition = Cairo1ContractDefinition(contractCode)
+        val contractCasmDefinition = CasmContractDefinition(casmCode)
+        val nonce = account.getNonce().send()
+
+        val declareTransactionPayload = account.signDeclare(
```

Comment:
So let me just ask - I suppose calculating hashes is exactly the same for cairo 2 as it was for cairo 1, right?

Reply:
I wasn't able to find any info about changes to hash calculations and the test with declaring 2.0.0 contract passes.